sys/fs/constfs/constfs.c

```diff
@@ -244,18 +203,6 @@ static ssize_t constfs_read(vfs_file_t *filp, void *dest, size_t nbytes)
     return nbytes;
 }
 
-static ssize_t constfs_write(vfs_file_t *filp, const void *src, size_t nbytes)
-{
-    DEBUG("constfs_write: %p, %p, %lu\n", (void *)filp, src, (unsigned long)nbytes);
-    /* Read only file system */
-    DEBUG("constfs_write: read only FS\n");
-    /* prevent warning: unused parameter */
-    (void) filp;
-    (void) src;
-    (void) nbytes;
-    return -EBADF;
```

Comment:
This is technically a change (vfs_write without a write returns -EINVAL), but I'd say it's for the better.